src/components/Select/Select.jsx

```diff
@@ -62,6 +62,10 @@ class Select extends React.Component {
       requiredSpan = <span className="form-required-span">(*Required)</span>;
     }
 
+    const ariaDescribedby =
+      [errorSpanId, this.props.ariaDescribedby || ''].join(' ').trim() ||
```

Comment:
We have the error id here as well :thinking:

Reply:
The `errorSpanId` is only set as a non-empty string when there is a validation error. The `aria-describedby` attributes does allow multiple ids as long as they are space separated.

Reply:
I think it's strange that we're including the `errorSpanId` in the `Select` and `NumberInput` components, but not in `Date`. It seems like the id is there:

https://github.com/department-of-veterans-affairs/component-library/blob/59f39ca75d652ecbaa0389b590366275e598cd6a/src/components/Date/Date.jsx#L88

Reply:
Hmm, you're right... I'll add the `errorSpanId` to the `ariaDescribedby` prop passed to the elements... on Monday when I work on it again 😸